ArduCopter/version.h

```diff
@@ -1,12 +1,12 @@
 #pragma once
 
-#include "ap_version.h"
```

Comment:
Yeah, it is transitively included; trying to make the AP_Common library a one-stop-shop for the version information.